ui/src/plugins/vumc/biovu.tsx

```diff
@@ -55,10 +55,16 @@ interface Data {
   (
     underlaySource: UnderlaySource,
     c: CommonSelectorConfig,
-    dataEntry?: DataEntry
+    dataEntries?: DataEntry[]
```

Comment:
is array support for future functions? i.e. only 1 entry is supported in this code